spring-boot-apps/miranum-deployment/pom.xml

```diff
@@ -63,7 +59,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>${maven.compiler.plugin.version}</version>
+                <version>3.11.0</version>
```

Comment:
Welchem Pattern folgst du hier, dass du manche versionen in die props rein nimmst und andere nicht?